src/views/DefineCompany.vue

```diff
@@ -28,27 +28,56 @@
         </div>
       </div>
     </v-card>
+    </section>
+    <p></p>
+    <section>
+      <header>
+        <h2>3. Business Contact Information</h2>
+      </header>
+      <BusinessContactInfo
+      :initialValue = businessContact
+      :isEditing = true
+      :showErrors = showErrors
+      @contactInfoChange = "onBusinessContactInfoChange($event)"
+      @contactInfoFormValidityChange = "onBusinessContactFormValidityChange($event)"/>
+    </section>
   </div>
 </template>
 
 <script lang="ts">
 // Libraries
 import { Component, Vue } from 'vue-property-decorator'
-import { Getter, Action } from 'vuex-class'
+import { Getter, Action, State } from 'vuex-class'
 
 // Interfaces
-import { GetterIF, ActionBindingIF } from '@/interfaces'
+import { GetterIF, ActionBindingIF, BusinessContactIF } from '@/interfaces'
 
 // Enums
 import { EntityTypes } from '@/enums'
 
-@Component({})
+// Components
+import { BusinessContactInfo } from '@/components/DefineCompany'
+
+@Component({
+  components: {
+    BusinessContactInfo
+  }
+})
 export default class DefineCompany extends Vue {
+  // State
+  @State(state => state.stateModel.defineCompanyStep.businessContact)
+  readonly businessContact!: BusinessContactIF
```

Comment:
Once again, i think this is pretty cool! Is this to make accessing the sub properties of the model easier?